.travis.yml

```diff
@@ -1,17 +1,27 @@
 dist: xenial  # required for Python >= 3.7 (travis-ci/travis-ci#9069)
 language: python
+
 python:
   - 3.4
```

Comment:
3.4 is end of life.

Reply:
Done in https://github.com/asciidoc/asciidoc-py3/pull/77